thingif/src/main/java/com/kii/thingif/clause/trigger/AndClauseInTrigger.java

```diff
@@ -23,18 +25,29 @@ public AndClauseInTrigger(@NonNull TriggerClause ...clauses){
 
     @Override
     public List<TriggerClause> getClauses() {
-        return this.clauses;
+        return new ArrayList<>(this.clauses);
     }
 
     @Override
-    public void addClause(@NonNull TriggerClause clause) {
+    public AndClauseInTrigger addClause(@NonNull TriggerClause clause) {
         this.clauses.add(clause);
+        return this;
     }
 
     @Override
     public JSONObject toJSONObject() {
```

Comment:
Exactly:+1:

If we would like to provide a feature to generate JSON object to developers, We should add this but if this method is for internal use, We can select GSON.